src/main/java/com/refinedmods/rangedpumps/blockentity/PumpBlockEntity.java

```diff
@@ -133,7 +133,7 @@ public void tick() {
         }
 
         if ((RangedPumps.SERVER_CONFIG.getSpeed() == 0 || (ticks % RangedPumps.SERVER_CONFIG.getSpeed() == 0)) && getState() == PumpState.WORKING) {
-            if (currentPos == null || currentPos.getY() == 0) {
+            if (currentPos == null || currentPos.getY() == -64) {
```

Comment:
Isn't there a constant for this somewhere?

Reply:
Fixed